code-challenges/401challenges/src/test/java/challenges/LinkedListTest.java

```diff
@@ -65,4 +65,61 @@
         assertEquals(actual, expected);
     }
 
+    @Test public void testLinkedListAppendMethod(){
+        LinkedList libraryForTest = new LinkedList();
+        libraryForTest.insert(10);
```

Comment:
It would have been better to have only used `.append()` here instead of also using `.insert()`